lib/core/network/api/aws_amplify/aws_authenticated_request.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:amazon_cognito_identity_dart_2/cognito.dart';
+import 'package:amazon_cognito_identity_dart_2/sig_v4.dart';
+import 'package:dio/dio.dart';
+import 'package:hypha_wallet/core/network/networking_manager.dart';
+
```

Comment:
should this request use our Http instance?

Reply:
I am not sure. Could try. Maybe a follow up improvement.